lang/src/main/kotlin/org/partiql/lang/ast/IsListParenthesizedMeta.kt

```diff
@@ -0,0 +1,20 @@
+package org.partiql.lang.ast
+
+/**
+ * The [IsListParenthesizedMeta] is used to distinguish the two types of [org.partiql.lang.domains.PartiqlAst.Expr.List]
+ * syntax's:
+ *  1. [[ a, b, c ]]
```

Comment:
Why the double `[[` and `]]`?

Reply:
It escapes the reserved square brackets -- here's a snippet of the generated HTML:
```
syntax's:</p><ul><li><p>[ a, b, c ]</p></li><li><p>( a, b, c )</p></li></ul><p>
```